src/screens/OtherProfile/index.js

```diff
@@ -298,23 +398,16 @@ const OtherProfile = () => {
       );
     };
 
-    if (isBlocked) return <></>;
+    if (blockStatus.blocked) return <></>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The block status of a user is now determined by `blockStatus.blocked` instead of `isBlocked`. Make sure that this new variable correctly reflects the block status.
```diff
+ if (blockStatus.blocked) return <></>;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->